src/index.html

```diff
@@ -7,8 +7,52 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img
+          src="./images/logo.png"
+          alt="logo"
+          class="logo__icon"
+        >
+      </a>
+
+      <nav>
+        <ul class="nav__bar">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              data-qa="hover"
+              class="nav__link"
+            >Laptops & Computers</a>
```

Comment:
please check the spaces
![image](https://user-images.githubusercontent.com/22824431/229493228-62ad3925-3813-4ed1-84e6-43904f02d31d.png)
```suggestion
            <a
              href="#"
              data-qa="hover"
              class="nav__link"
            >
              Laptops & Computers
            </a>
```